tikv-client/src/main/java/com/pingcap/tikv/PDClient.java

```diff
@@ -66,6 +69,24 @@
   private ScheduledExecutorService service;
   private List<URI> pdAddrs;
 
+  public void scatterRegion(TiRegion left) {
```

Comment:
what's the difference between `RegionStoreClient.splitRegion` and `PDClient. scatterRegion`?

Reply:
` RegionStoreClient.splitRegion` sends reequest to tikv split region. 
`PDClient. scatterRegion` send scatter region request to pd because pd has region scheduler which can balance regions.

Reply:
sync call or async call?
what if one call success while another fail?

Reply:
it's sync call. After the split, the region does not contain any data. Scatter simples shuffle regions into mulitiple TiKV instance.